mcp/housekeeping/cryo_tauhk.c

```diff
@@ -0,0 +1,974 @@
+            htr_sw_fpa_pwm_Addr = channels_find_by_name("htr_sw_fpa_pwm");
+            htr_sw_fpa_volts_Addr = channels_find_by_name("htr_sw_fpa_volts");
+            htr_uc_hsw_dac_Addr = channels_find_by_name("htr_uc_hsw_dac");
+            htr_uc_hsw_overcurrent_monitor_Addr = channels_find_by_name("htr_uc_hsw_overcurrent_monitor");
+            htr_uc_hsw_pwm_Addr = channels_find_by_name("htr_uc_hsw_pwm");
+            htr_uc_hsw_volts_Addr = channels_find_by_name("htr_uc_hsw_volts");
+            htr_uc_pump_dac_Addr = channels_find_by_name("htr_uc_pump_dac");
+            htr_uc_pump_overcurrent_monitor_Addr = channels_find_by_name("htr_uc_pump_overcurrent_monitor");
+            htr_uc_pump_pwm_Addr = channels_find_by_name("htr_uc_pump_pwm");
+            htr_uc_pump_volts_Addr = channels_find_by_name("htr_uc_pump_volts");
+            htr_vcs1_dac_Addr = channels_find_by_name("htr_vcs1_dac");
+            htr_vcs1_overcurrent_monitor_Addr = channels_find_by_name("htr_vcs1_overcurrent_monitor");
+            htr_vcs1_pwm_Addr = channels_find_by_name("htr_vcs1_pwm");
+            htr_vcs1_volts_Addr = channels_find_by_name("htr_vcs1_volts");
+            htr_vcs2_dac_Addr = channels_find_by_name("htr_vcs2_dac");
+            htr_vcs2_overcurrent_monitor_Addr = channels_find_by_name("htr_vcs2_overcurrent_monitor");
+            htr_vcs2_pwm_Addr = channels_find_by_name("htr_vcs2_pwm");
+            htr_vcs2_volts_Addr = channels_find_by_name("htr_vcs2_volts");
+            rtd_1_2_logdac_Addr = channels_find_by_name("rtd_1_2_logdac");
+            rtd_1_3_logdac_Addr = channels_find_by_name("rtd_1_3_logdac");
+            rtd_1_4_logdac_Addr = channels_find_by_name("rtd_1_4_logdac");
+            rtd_1_5_logdac_Addr = channels_find_by_name("rtd_1_5_logdac");
+            rtd_1_6_logdac_Addr = channels_find_by_name("rtd_1_6_logdac");
+            rtd_1_7_logdac_Addr = channels_find_by_name("rtd_1_7_logdac");
+            rtd_4he_head_logdac_Addr = channels_find_by_name("rtd_4he_head_logdac");
+            rtd_ic_head_logdac_Addr = channels_find_by_name("rtd_ic_head_logdac");
+            rtd_lw_fpu_1k_logdac_Addr = channels_find_by_name("rtd_lw_fpu_1k_logdac");
+            rtd_lw_fpu_250_logdac_Addr = channels_find_by_name("rtd_lw_fpu_250_logdac");
+            rtd_lw_fpu_350_logdac_Addr = channels_find_by_name("rtd_lw_fpu_350_logdac");
+            rtd_strap_inter_logdac_Addr = channels_find_by_name("rtd_strap_inter_logdac");
+            rtd_sw_fpu_1k_logdac_Addr = channels_find_by_name("rtd_sw_fpu_1k_logdac");
+            rtd_sw_fpu_250_logdac_Addr = channels_find_by_name("rtd_sw_fpu_250_logdac");
+            rtd_sw_fpu_350_logdac_Addr = channels_find_by_name("rtd_sw_fpu_350_logdac");
+            rtd_uc_head_logdac_Addr = channels_find_by_name("rtd_uc_head_logdac");
+            first_time = 0;
+        }
+        SET_SCALED_VALUE(diode_4he_film_enabled_chop_Addr, hk_data_one.diode_4he_film_enabled_chop);
```

Comment:
why incharge at all, don 't both comps get the data and we dont need to put it on the sharing server?